File: Python/Processor.py
class Processor:
    __name = "" # menampung nama processor
    __price = 0 # menampung harga

    # constructor
    def __init__(self, name = "", price = 0):
        self.__name = name
        self.__price = price

    # setter and getter
    def setName(self, name):
        self.__name = name
    
    def getName(self):
        return self.__name

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

    # method
    def printProcessor(self):
        print("Processor Name   : " + str(self.getName()))
        print("Processor Price  : Rp " + str(self.getPrice()) + ",-")

File: PHP/Processor.php
<?php

class Processor {
    private $name;  // menampung nama processor
    private $price; // menampung harga

    // constructor
    function __construct($name, $price) {
        $this->name = $name;
        $this->price = $price;
    }

    // setter and getter
    function setName($name) {
        $this->name = $name;
    }

    function getName() {
        return $this->name;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    // method
    function printProcessor() {
        echo "Processor Name    : ". $this->getName(). "<br/>";
        echo "Processor Price   : Rp ". $this->getPrice(). ",-<br/>";
    }
}

?>